masonry/src/widget/widget_state.rs

```diff
@@ -124,12 +119,11 @@ pub struct WidgetState {
     pub(crate) text_registrations: Vec<TextFieldRegistration>,
 
     // --- STATUS ---
-    // `true` if one of our ancestors is disabled (meaning we are also disabled).
-    pub(crate) ancestor_disabled: bool,
 
-    // `true` if this widget has been explicitly disabled.
-    // A widget can be disabled without being *explicitly* disabled if an ancestor is disabled.
+    // This widget has been disabled.
```

Comment:
This should probably be a doc-comment
```suggestion
    /// This widget has been disabled.
```